pkg/kubelet/stats/cri_stats_provider_windows.go

```diff
@@ -20,104 +20,102 @@ limitations under the License.
 package stats
 
 import (
-	"fmt"
 	"time"
 
+	"k8s.io/klog/v2"
+
 	"github.com/Microsoft/hcsshim"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/klog/v2"
 	statsapi "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
 )
 
+// hcsShimInterface creates an interface that allows for testing the logic without needing to create a container
+type hcsShimInterface interface {
+	HNSListEndpointRequest() ([]hcsshim.HNSEndpoint, error)
+	GetHNSEndpointStats(endpointName string) (*hcsshim.HNSEndpointStats, error)
+	Now() time.Time
+}
+
+// realHcsShim exposes the required functionality for hcsshim in this scenario
+type realHcsShim struct{}
+
+func (s realHcsShim) HNSListEndpointRequest() ([]hcsshim.HNSEndpoint, error) {
+	return hcsshim.HNSListEndpointRequest()
+}
+
+func (s realHcsShim) GetHNSEndpointStats(endpointName string) (*hcsshim.HNSEndpointStats, error) {
+	return hcsshim.GetHNSEndpointStats(endpointName)
+}
+
+func (s realHcsShim) Now() time.Time { return time.Now() }
+
 // listContainerNetworkStats returns the network stats of all the running containers.
 func (p *criStatsProvider) listContainerNetworkStats() (map[string]*statsapi.NetworkStats, error) {
-	containers, err := hcsshim.GetContainers(hcsshim.ComputeSystemQuery{
-		Types: []string{"Container"},
-	})
+	shim := newHcsShim(p)
+
+	endpoints, err := shim.HNSListEndpointRequest()
 	if err != nil {
+		klog.ErrorS(err, "Failed to fetch current endpoints endpoints")
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to fetch current endpoints", endpoints)
```
Was this the intended behavior here?

Reply:
it was duplicated by mistake, will fix this and the other typo

Reply:
updated